goal_src/jak2/engine/target/target-darkjak.gc

```diff
@@ -189,9 +189,9 @@
       (set! (-> a3-7 w) 1.0)
       (t9-14 a0-43 (the-as vector a1-11) (the-as quaternion a2-8) a3-7)
       )
-    ;; (set-darkjak-texture-morph! f30-0)
+    (set-darkjak-texture-morph! f30-0)
```

Comment:
this changed (I think it will still crash?)

Reply:
Does not crash.